packages/ui/src/components/AccountDisplay.tsx

```diff
@@ -71,11 +71,13 @@ const AccountDisplay = ({
 
   return (
     <div className={className}>
-      <IdenticonBadge
-        badge={badge}
-        address={encodedAddress}
-        size={iconSize}
-      />
+      <span data-cy="icon-identity">
+        <IdenticonBadge
+          badge={badge}
+          address={encodedAddress}
+          size={iconSize}
+        />
+      </span>
```

Comment:
@Tbaut I tried everything to avoid adding a span but this is the only thing that would work here (and in the other place where I added one)

Reply:
This is something we can most probably work around. I'll check it out.